span.go

```diff
@@ -181,3 +184,44 @@ func (r *spanS) SetTag(key string, value interface{}) ot.Span {
 func (r *spanS) Tracer() ot.Tracer {
 	return r.tracer
 }
+
+// sendOpenTracingLogRecords converts OpenTracing log records that contain errors
+// to Instana log spans and sends them to the agent
+func (r *spanS) sendOpenTracingLogRecords() {
+	for _, lr := range r.Logs {
+		for _, lf := range lr.Fields {
+			if isErrorLogField(lf) {
+				r.sendOpenTracingLogRecord(lr)
+				break
+			}
+		}
+	}
+}
+
+func (r *spanS) sendOpenTracingLogRecord(lr ot.LogRecord) {
+	buf := bytes.NewBuffer(nil)
+
+	enc := newOpenTracingLogEncoder(buf)
+	for _, lf := range lr.Fields {
+		lf.Marshal(enc)
+		buf.WriteByte(' ')
+	}
+
+	r.tracer.StartSpan(
+		"log.go",
+		ot.ChildOf(r.context),
+		ot.StartTime(lr.Timestamp),
+		ot.Tags{
+			"log.level":   "ERROR",
+			"log.message": strings.TrimSpace(buf.String()),
+		},
+	).FinishWithOptions(
+		ot.FinishOptions{
+			FinishTime: lr.Timestamp,
+		},
+	)
+}
+
+func isErrorLogField(lf otlog.Field) bool {
```

Comment:
```suggestion
func (r *spanS) isErrorLogField(lf otlog.Field) bool {
```
This function is used only inside spanS, what do you think if we will make a method of spanS?

Reply:
I see your point to use the receiver as a namespace, however this helper function is relevant to any `opentracing.Field` and I believe there is no need to restrict its usage to our spans only. Do you see any disadvantages in this approach?

Reply:
In this particular case, I don't think there are any. It is more about "cohesion" and responsibility in general. Having a lot of helpers function may lead to creating the packages/modules like "util", which is often considered a not best practice. Nevertheless, I see your point, and being pragmatic keeping as it is is absolutely fine for me. 👍